source/dofs/dof_renumbering.cc

```diff
@@ -1191,50 +1191,65 @@ namespace DoFRenumbering
   namespace
   {
     // helper function for hierarchical()
-
-// Note that this function only works for active dofs.
-    template <int dim, class iterator>
+    //
+    // Note that this function only works for locally active dofs.
```

Comment:
I think the meaning of that comment was "active" in the sense of "not level dofs", but I am not sure.

Reply:
Ah, hm. I'll just take the comment out then.